frontend/workflows/projectSelector/src/project-selector.tsx

```diff
@@ -80,7 +86,12 @@ const ProjectSelector = () => {
 
   const { updateSelected } = useDashUpdater();
 
-  const [state, dispatch] = React.useReducer(selectorReducer, initialState);
+  const localState = window.localStorage.getItem(STORAGE_KEY);
+  const finalizedState = localState ? JSON.parse(localState) : {};
+  const [state, dispatch] = React.useReducer(
+    selectorReducer,
+    _.merge(initialState, finalizedState)
```

Comment:
how will we ensure there aren't backwards compatibility issues if we update the structure of the state or storage? i think we need to validate that.